templates/messages/issue_update.html

```diff
@@ -12,6 +12,9 @@
 {% elif changes.assignees %}
     {{ assignee_changes(changes.assignees.added, changes.assignees.removed) }}
     {{ issue_or_merge_link(object_attributes) }}
+{% elif changes.reviewers %}
+    {{ assignee_changes(changes.reviewers.added, changes.reviewers.removed) }}
```

Comment:
This should have a new macro, requesting a review and assigning are different things